assets/src/stories-editor/blocks/amp-story-page/edit.js

```diff
@@ -97,7 +99,7 @@ class PageEdit extends Component {
 	 * @param {Object} media.image      Media image object.
 	 * @param {string} media.image.src  Media image URL
 	 */
-	onSelectMedia = ( media ) => {
+	onSelectMedia( media ) {
```

Comment:
This should be changed back to use class properties

```suggestion
	onSelectMedia = ( media ) => {
```